FlowCrypt/Functionality/Services/Organisational Rules Service/OrganisationalRulesService.swift

```diff
@@ -14,10 +15,14 @@ protocol OrganisationalRulesServiceType {
     func fetchOrganisationalRules(for email: String) -> Promise<OrganisationalRules>
 
     func getSavedOrganisationalRulesForCurrentUser() -> OrganisationalRules
+
+    func getEmailKeyManagerPrivateKeys() -> Promise<OrganisationalRulesService.GetEKMKeysResult>
 }
 
 final class OrganisationalRulesService {
 
+    typealias GetEKMKeysResult = (keys: [DecryptedPrivateKey], urlString: String?)
```

Comment:
I think we don't need a separate `GetEKMKeysResult` just to additionally contain URL. URL for the toast can be gotten separately, from org rules directly again.

Better reuse `DecryptedPrivateKeysContainer` here but call it `EkmPrivateKeysGetResponse`